### PR TITLE
Replace current bootstrap CSS

### PR DESCRIPTION
**Pull request by juliantoledo**
I am removing the old bootstrap CSS and adding our custom one.

### CODE REVIEW

File: views/includes/head.hbs [hunk 1]
```diff
@@ -14,5 +14,183 @@
...
+  text-decoration: none;
+  display: block;
+  width: 100%;
+  height: 100%;
+}
+
+.media {
+  border: 2px solid lightgray;
+  padding: 8px 4px 4px 8px;
+  text-overflow: ellipsis;
```
Comment (andreban) [inline views/includes/head.hbs]: Can remove text-overflow: ellipsis for now, as it's not working. Maybe adding "overflow: hidden;" so it won't show outside the box and ellipsizing the headers and texts elements.

**Comment by andreban** [comment]
It may be worth having the CSS inside it's own include in order to make it easier adding it to AMPs.

**Comment by juliantoledo** [comment]
I did author these commits.

**Comment by juliantoledo** [comment]
I finished the clean up and fixed @andreban comment.

**Comment by juliantoledo** [comment]
@andreban I actually added some of your commits too. hum.

**Comment by andreban** [comment]
LGTM

**Pull request merged**